sdk-incubator-lib/src/main/java/cash/z/ecc/android/sdk/internal/Twig2.kt

```diff
@@ -15,7 +15,7 @@ import java.util.Locale
  * or usage in production.
  */
 @Suppress("TooManyFunctions")
-object Twig {
+object Twig2 {
```

Comment:
Twig2 is not the name I want here—it avoids a naming conflict with the original Twig class.

This might be a good time to replace the original Twig calls so we have a single simplified logging solution.

Reply:
Yes, I'm in favour of unify these logging components into one and solve it once for all.